### DISCUSSION TITLE
flytectl update launchplan, the schedule is not activated yet

### DISCUSSION THREAD

**Discussion by drleowang** [flyteorg/flyte#2715] [Q&A]
I installed Flyte and enable Cloudwatch event and SQS to set schedule for a worflow, I created launchplan in python code, and register workflow and launchplan, they works well. 

I run "flytectl update launchplan  -p project -d dev workflow_sqs  --activate --version  e1f762cb1bb6*******1"  and I saw a cloudwatch event was created in AWS. But the workflows on Flyte platform did not run on Schedule yet.(If I just modified cloudwatch event and updated in AWS, then the workflows will run on schedule ) I am wondering why the workflow did not run on schedule automatically??

Thanks a lot!

**Comment by samhita-alla** [comment]
cc: @pmahindrakar-oss

**Comment by kumare3** [comment]
@drleowang why did you not use the native scheduler? Any reason. You do not need sqs or cloudwatch to schedule with native scheduler, it should just work
